content/software-testing-the-hardware-software-interface-part-2.md

```diff
@@ -0,0 +1,417 @@
+---
+title: "Embedded Software Testing: Practical Continuous Integration with Hardware in the Loop (Part 2)"
+date: 2024-11-03
+author: Leonardo Held, Torizon Developer
+draft: false
+
+tags: ["Testing", "Embedded", "Linux", "Embedded"]
+
+abstract: A series about Embedded Software Testing.
+
+image: /soave-gate.jpg
+---
+
+This is the second installment in the embedded software testing series. This one will be about the architecture of a 
+system to test embedded software. We'll briefly cover the what is usually called a "dual-targetting" approach as well.
+
+### Theory
+
+In the previous installment we talked about the different levels of software testing and the last one was "end-to-end
```

Comment:
> "In the previous installment we talked about the different levels of software testing and the last one was 'end-to-end testing'"

Too long without commas, it would be better to split.

Reply:
Sure, fixed.